### PR TITLE
[RFR] Landing page tweaks

### PR DESCRIPTION
**Pull request by eschaefer**
- add twitter/og tags
- add favicons
- mobile header image size tweaks

### CODE REVIEW

**Comment by eschaefer** [comment]
ping @edenspiekermann/devs

File: web/templates/layout/app.html.eex [hunk 1]
```diff
@@ -1,3 +1,8 @@
+<%
+  title = "POST Bot for Slack"
```
Comment (nburkley) [inline web/templates/layout/app.html.eex]: The strings should be pulled out and handled with gettext. The Logo url should be in a view (similar to a rails view helper). Just like with rails, we shouldn't be defining variables in the template.

Comment (nburkley) [reply web/templates/layout/app.html.eex]: I'll add a commit to look after this

**Review by nburkley — APPROVED** [review]
ping @edenspiekermann/css

**Comment by nburkley** [comment]
@eschaefer I've pulled out those strings and fixed the static assts. We should be using the `static_asset` helper for all static assets and for assets that need to be at root level we need to set them to pass through in `endpoint.ex`

I'm only figuring this out as I go along!

File: web/templates/layout/app.html.eex [hunk 2]
```diff
@@ -6,9 +6,26 @@
 <head>
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-  <title>POST Bot for Slack</title>
-  <meta name="description" content="A Slack bot to get valuable feedback on your articles.">
+  <title><%= gettext "Default Title" %></title>
+  <meta name="description" content="<%= gettext "Default Description" %>">
```
Comment (eschaefer) [inline web/templates/layout/app.html.eex:10]: Great, good to know

**Comment by eschaefer** [comment]
@nburkley good call, thanks!

**Pull request merged**